pwa/src/components/logs/logTable.tsx

```diff
@@ -196,9 +222,8 @@ export default function LogTable({ id = null }) {
                     aria-labelledby="logRequest-tab"
                   >
                     <h5 className="utrecht-heading-5 utrecht-heading-5--distanced mt-3">Method: {log.requestMethod && log.requestMethod}</h5>
-                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced">Path info: {log.requestPathInfo && log.requestPathInfo}</h5>
-                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced">Languages: {log.requestLanguages && log.requestLanguages}</h5>
-                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced">Status: {log.requestStatus && log.requestStatus} {log.requestStatusCode && log.requestStatusCode}</h5>
+                    <h5 className="utrecht-heading-5 utrecht-heading-5--distanced mt-1">Path info: {log.requestPathInfo && log.requestPathInfo}</h5>
```

Comment:
Haakt in op bovenstaande comment:

Ik denk dat je hier een check doet of `log.requestMethod` bestaat en wanneer die bestaat je hem wilt tonen. Een makkelijkere manier hiervoor zou zijn `log?.requestMethod`.